<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="PHPUnit_Framework_OutputError" namespace="" name="PHPUnit_Framework_OutputError" abstract="false" final="false" start="17" end="19">
  <file path="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/phpunit/phpunit/src/Framework" file="OutputError.php" realpath="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/phpunit/phpunit/src/Framework/OutputError.php" size="487" time="2015-12-17T18:44:10+00:00" unixtime="1450377850" sha1="f4ffcf3567da47367f4a3e83aba06f4845497aed" relative="vendor/phpunit/phpunit/src/Framework/OutputError.php"/>
  <docblock>
    <description compact="Extension to PHPUnit_Framework_AssertionFailedError to mark the special case of a test that printed output."/>
    <since value="Class available since Release 3.6.0"/>
  </docblock>
  <extends full="PHPUnit_Framework_AssertionFailedError" namespace="" name="PHPUnit_Framework_AssertionFailedError"/>
</class>
